rest/server.go

```diff
@@ -0,0 +1,66 @@
+package rest
+
+import (
+	"awesomeProject3/service"
+	"encoding/json"
+	"log"
+	"net/http"
+)
+
+type container struct {
+	Getter string `json:"getter"`
+	Value  string `json:"value"`
+}
+
+type httpServer struct {
+	ssService service.SecretService
+}
+
+func NewHttpServer(sr service.SecretService) httpServer {
+	return httpServer{
+		ssService: sr,
+	}
+}
+
+func (sr *httpServer) Start() {
+	http.HandleFunc("/", sr.homePage)
+	log.Fatal(http.ListenAndServe(":10000", nil))
+}
+
+func (sr *httpServer) homePage(w http.ResponseWriter, r *http.Request) {
```

Comment:
Refactor into two different handlers. I don't see usage of `github.com/go-chi/chi`